### PR TITLE
AIFBR-6791: Consolidate All AIFabric Databases into one DB

### PR DESCRIPTION
**Pull request by rajivml**
- AIFBR-6791
- Consolidate all AIFabric databases into one DB
- Change are backward compatible i.e. the users already using multiple databases shouldn't get affected and for SaaS we will be still using multiple databases

### CODE REVIEW

File: database/createDatabases.ps1 [hunk 1]
```diff
@@ -117,11 +124,33 @@ if(!$installed){
...
+    GO
+    "
+}
+else {
 $sqlcommand = "
-GO
-CREATE LOGIN $dbuser WITH PASSWORD=N'{{pwd}}'
-GO
-"
+    GO
```
Comment (mohitseth-nitrr) [inline database/createDatabases.ps1:144]: This would make default value as "Y" which seems wrong to me

**Review by mohitseth-nitrr — DISMISSED** [review]
Lets remove platform/aks folder altogether as these are already hosted on download.uipath.com ... it will create confusion if these two become out of sync

File: database/createDatabases.ps1 (hunk 1 above)
Comment (rajivml) [reply database/createDatabases.ps1:144]: Nope, if the argument is not passed, it's getting set to "N" at the start of the script 

if($singleDatabase.Length -eq 0){
    write-host "creating multiple databases"
    $singleDatabase = "N"
}

**Comment by rajivml** [comment]
> Lets remove platform/aks folder altogether as these are already hosted on download.uipath.com ... it will create confusion if these two become out of sync

will move it in an other PR by fixing all the references

**Review by aniket202 — APPROVED** [review]
LGTM

**Review by Mdirfan6 — APPROVED** [review]
lgtm

**Pull request merged**